docs/Developers/Git_Basics.rst

```diff
@@ -73,11 +73,34 @@ Now you should add a remote to your local machine so that you can ``pull`` and
   **myfork** is just a name we used for simplicity. You can
   name it however you want.
 
+The next thing you should know before getting down to work is how to check on
+the changes you've made to your project, and your current branch. The
+ability to check your current branch is also extremely important, as you'll see
+in the next section. The command to check this information is:
+
+::
+
+    $ git status
+
+Before we move onto the next section, you need to know about a very important
+branch called master. Master is the default branch that git checkouts for you
+when you cloned the repository. It's our policy here at coala to never develop
+on the master branch. This is why we create new branches, which leads us to the
+next section.
+
 Creating a new branch
 ---------------------
 
 To start working on an issue, you first need to create a new branch where you
-will work.
+will work. Do not change files when you are on your master branch. If you
+submit a Pull Request from your master branch, you are publicly admitting
+that you do not understand git, and have not read this documentation yet.
```

Comment:
The following is a bit aggressive in tone for a newcomer document:
`you are publicly admitting that you do not understand git, and have not read this documentation yet`

It's probably good enough to just say that coala developers are likely to reject your PR. 
How about this instead?
`If you submit a Pull Request from your master branch, coala developers are likely to reject your work (even if it is a good patch).`